client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -30,9 +34,16 @@ export default function Dashboard(): JSX.Element {
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
       <CssBaseline />
+
       <Grid item className={classes.drawerWrapper}>
         <ChatSideBanner loggedInUser={loggedInUser} />
       </Grid>
+      <Grid>
+        <Button color="secondary" onClick={() => setUploadPhoto(!uploadPhoto)}>
+          Profile Photos
+        </Button>
+      </Grid>
+      <Grid>{uploadPhoto && <UploadPhoto />}</Grid>
```

Comment:
Let's either remove this testing code from the PR or leave a `TODO` comment to remind about removing this.